### PR TITLE
Jseldess/statement privileges

### LINKED ISSUE #225: Privileges per statement
Add required privileges for each SQL statement.

### PR DESCRIPTION
**Pull request by jseldess**
Documented required privileges for each supported SQL statement. Closes #225.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/230)

### CODE REVIEW

**Comment by jseldess** [comment]
@petermattis, if marc's going to be out for a while, anyone else I should ask to review?

File: create-index.md [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/create_index.html %}
 
-## Privileges
+## Required Privileges
+
+The user must have the `CREATE` privilege on the table.
```
Comment (mberhault) [inline create-index.md]: the `ALTER` documentation links to privilege.htm. I think we should either do that everywhere, or nowhere, but we should be consistent.

File: drop-database.md [hunk 2]
```diff
@@ -1,19 +1,66 @@
...
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/drop_database.html %}
 
-## Privileges
+## Required Privileges
+
+The user must have the `DROP` [privilege](privileges.html) on the database and all included tables.
```
Comment (mberhault) [inline drop-database.md]: how about: "all tables in the database" instead? just a nit, feel free to ignore.

File: set-transaction.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/set_transaction.html %}
 
-## Privileges
+## Required Privileges
+
+No privileges are required to set the transaction isolation level and/or   priority.
```
Comment (mberhault) [inline set-transaction.md]: a few too many spaces between "and/or" and "priority"

**Comment by mberhault** [comment]
LGTM. Very nice. My main comment is about being consistent with the link to `privileges.html` from all `required privileges` sections that list a specific one.

**Comment by jseldess** [comment]
Review status: 0 of 35 files reviewed at latest revision, 3 unresolved discussions.

---

*[create-index.md, line 18 \[r3\]](https://reviewable.io:443/reviews/cockroachdb/docs/230#-KFkPfpxo2QEluPz8hDl-r3-18:-KFtEzy8e8MLJiOZWZNa:1151043765) ([raw file](https://github.com/cockroachdb/docs/blob/3093446802fcc650ac0c3223b8f821a9f8cfd6b6/create-index.md#L18)):*
Good point. Making a bunch of updates with next commit.

---

*[drop-database.md, line 16 \[r3\]](https://reviewable.io:443/reviews/cockroachdb/docs/230#-KFkPfpyjLpsR04rNwXW-r3-16:-KFtFG_8QuZ17XdP9MUT:464585087) ([raw file](https://github.com/cockroachdb/docs/blob/3093446802fcc650ac0c3223b8f821a9f8cfd6b6/drop-database.md#L16)):*
Yep, going with "on the database and on all tables in the database."

---

*[set-transaction.md, line 16 \[r3\]](https://reviewable.io:443/reviews/cockroachdb/docs/230#-KFkPfq-QFBuKqwS3qlL-r3-16:-KFtFMr_8omRK2myYQv6:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/3093446802fcc650ac0c3223b8f821a9f8cfd6b6/set-transaction.md#L16)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/230)*

**Comment by jseldess** [comment]
Thanks for the review, @mberhault. Addressed all your concerns.

**Pull request merged**